WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -1705,6 +1792,25 @@ local function Constructor()
   expand:SetScript("OnEnter", function() Show_Tooltip(button, expand.title, expand.desc) end);
   expand:SetScript("OnLeave", Hide_Tooltip);
 
+  local update
+  if WeakAurasWagoUpdate then
+    update = CreateFrame("BUTTON", nil, button);
+    button.update = update
+    update.disabled = true;
+    update.func = function() end;
+    update:SetNormalTexture("interface\\minimap\\rotating-minimapcorpsearrow.blp");
```

Comment:
You can ask Exality if he wants to create a icon, I think a wago Icon with a download arrow might work pretty well.